appholder/src/main/java/com/ul/ims/gmdl/appholder/transfer/TransferManager.kt

```diff
@@ -0,0 +1,143 @@
+package com.ul.ims.gmdl.appholder.transfer
+
+import android.content.Context
+import android.os.Build
+import android.util.Log
+import android.view.View
+import androidx.core.content.ContextCompat
+import androidx.core.util.component1
+import androidx.core.util.component2
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.security.identity.*
+import com.ul.ims.gmdl.appholder.util.TransferStatus
+import java.util.concurrent.Executor
+
+class TransferManager private constructor(private val context: Context) {
```

Comment:
I would expect this to take a `Document` instance... that way you don't have to pass `IdentityCredentialStore` to `startPresentation()` nor pass `String credentialName` to `sendCredential()` below...

Reply:
It is using Document object now